docs/contribute/contribution_guidelines.rst

```diff
@@ -70,6 +70,17 @@ The testing for functions and classes should at least contain tests for
 
 In case of deprecations, it must also be tested if the deprecated feature is properly removed (see `tests/test_deprecations.py` for examples).
 
+Testing plotting functions
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The above also holds for plotting functions, however, in this case it is required that the plots generate baseline and test images. The following documents the current best practice. See `test_plot.py <https://github.com/pyfar/pyfar/blob/main/tests/test_plot.py>`_ for a code example.
+
+Baseline images show how an image produced by a given plot command (for example `pf.plot.time(pf.Signal[0, 1, 0], 1)`) should look like. Baseline images must be created when a new plot function or test is introduced, and they must be overwritten when the behavior of a plot function changes, e.g., because of a bug fix. Baseline images must be visually inspected as part of a pull request when they are created or changed.
+
+Test images show how an image produced by a given plot command currently looks like. Test images must be created each time the tests run. If the behavior of a pot function changes, the corresponding test image will change. In this case, the changed test image must be visually compared to the corresponding baseline image as part of a pull request.
+
+``pyfar.testing.plot_utils`` contains the function ``create_figure`` for creating a figure in a way that produces almost identical results across operating systems. The function ``save_and_compare`` is best practice for saving and comparing test plots automatically. However, the automatic comparison is usually disabled. It often fails because the plots generated by Matplotlib depend on the operating system. Different operating systems create offsets of a few pixels resulting in large differences between test and baseline plots.
```

Comment:
I'd suggest providing this information on the issue with automated comparison before introducing the baseline images.

Reply:
You mean, I should link it here: https://github.com/pyfar/spharpy/pull/96 ?